### PR TITLE
Feature/move everything from category continuation

### PR DESCRIPTION
**Pull request by juntereiner**
This is the continuation of https://github.com/Cidan/BetterBags/pull/329. Original author is @fiote.

Related to https://github.com/Cidan/BetterBags/issues/319 and https://github.com/Cidan/BetterBags/issues/118 .

### CODE REVIEW

File: frames/section.lua [hunk 1]
```diff
@@ -239,6 +242,74 @@ local function onTitleClickOrDrop(section)
   events:SendMessage('bags/FullRefreshAll')
 end
 
+---@param section Section
+local function onTitleRightClick(section)
```
Comment (Cidan) [inline frames/section.lua:246]: Can you make this a method on Section and not a local function? Part of the ethos of BetterBags is to expose things as an API where possible.

Capitalize the function name when you expose it. Thanks!

File: core/init.lua [hunk 2]
```diff
@@ -207,6 +207,32 @@ function addon:UpdateButtonHighlight()
   end
 end
 
+
+function addon:AtSendMail()
```
Comment (Cidan) [inline core/init.lua:211]: Can you move these out of the addon space and either into their own, new, module, or some existing module?

Do the same for all these new addon methods.

Thanks!

**Review by Cidan — CHANGES_REQUESTED** [review]
LGTM except two small things. See below and thank you!

File: frames/section.lua (hunk 1 above)
Comment (juntereiner) [reply frames/section.lua:246]: Sure, do you want the same thing for onTitleClickOrDrop ? Seems inconsistent.

File: core/init.lua (hunk 2 above)
Comment (juntereiner) [reply core/init.lua:211]: I can create a new module MovementFlow, do you want it in core or util folder ?

Comment (Cidan) [reply core/init.lua:211]: Let's do util. Thanks!

File: frames/section.lua (hunk 1 above)
Comment (Cidan) [reply frames/section.lua:246]: Yes, please! Thank you for catching that!

File: frames/section.lua [hunk 3]
```diff
@@ -31,6 +31,12 @@ local themes = addon:GetModule('Themes')
 ---@class GridFrame: AceModule
 local grid = addon:GetModule('Grid')
 
+---@class Database: AceModule
+local db = addon:GetModule('Database')
+
+---@class movementFlow: AceModule
```
Comment (Cidan) [inline frames/section.lua:37]: Module classes start with an upper case.

File: util/movementflow.lua [hunk 4]
```diff
@@ -0,0 +1,37 @@
+local addonName = ... ---@type string
+
+---@class BetterBags: AceAddon
+local addon = LibStub('AceAddon-3.0'):GetAddon(addonName)
+
+---@class Constants: AceModule
+local const = addon:GetModule('Constants')
+
+---@class movementFlow: AceModule
```
Comment (Cidan) [inline util/movementflow.lua:9]: Module classes start with upper case. Note, this is for the class name, not the variable.

**Review by Cidan — APPROVED** [review]

**Pull request merged**